src/components/Menu/MainNavigation.tsx

```diff
@@ -67,42 +64,59 @@ export default function MainNavigation({
               edge="start"
               color="inherit"
               sx={{ mr: 1 }}
-              onClick={handleMenu}
+              onClick={handleDrawerOpen}
             >
               <MenuIcon />
             </IconButton>
-            <SportsIcon />
+            <IconButton component={NavLink} to={tabs[1].value} color="inherit">
```

Comment:
please rewrite a bit router, create separate files `router.tsx` and `routes.tsx` and change all hardcoded values. Use for navigation same const as in router config. For inspiration:

<img width="742" alt="Screenshot 2024-05-19 at 11 06 06" src="https://github.com/proga-react-practice/xi-team-project/assets/103277184/f5f587d9-a855-4971-9616-c18b4f951153">